examples/qml_features/cpp/qabstractlistmodelcxx.h

```diff
@@ -10,6 +10,13 @@
 
 #include "rust/cxx.h"
 
+template<class T>
```

Comment:
we've been using `typename` elsewhere instead of `class`

Reply:
This file was now be removed anyway :partying_face: